pom.xml

```diff
@@ -239,31 +239,31 @@
         <dependency>
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-archiver</artifactId>
-            <version>3.4</version>
+            <version>4.2.2</version>
         </dependency>
         <dependency>
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
-            <version>2.6</version>
+            <version>2.7</version>
```

Comment:
@maciejurbanskisaucelabs 
The projects sets [minimum JDK to 7](https://github.com/saucelabs/ci-sauce/blob/master/pom.xml#L84-L85), however `commons-io:2.7` [requires JDK 8](http://commons.apache.org/proper/commons-io/index.html):
> Commons IO 2.7 requires a minimum of Java 8

Reply:
@valfirst do you require JDK 7? I think we can get it back to Commons IO 2.6 which requires JDK 7.

Reply:
@adambabik no, we don't require JDK 7, I just would like to highlight that JDK version should be bumped (or if you have JDK 7 users, this change should be reverted)